tests/unit/plugins/savedreplies.js

```diff
@@ -101,7 +101,7 @@ describe( 'Plugins', () => {
 						component.render();
 						component.isOpen = true;
 
-						expect( list.focus.callCount ).to.equals( 2 ); // #363
+						expect( list.focus.callCount ).to.equals( 1 ); // #363
```

Comment:
```suggestion
						expect( list.focus.callCount ).to.equals( 1 );
```

Reply:
So we made a full circle here 😂 Look at #395 

Since it again works as it should there's no need for a reference here.